lib/SIL/SILVerifier.cpp

```diff
@@ -108,6 +108,7 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
   SILFunctionConventions fnConv;
   Lowering::TypeConverter &TC;
   SILOpenedArchetypesTracker OpenedArchetypes;
+  SmallVector<SILDebugVariable, 16> DebugVars;
```

Comment:
VarInfo is StringRef + ArgNo, essentially. It is not heavy. And it is a verifier, which only runs on demand.